catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -541,8 +541,14 @@ public CallableSourceResponse(Source source, QueryRequest request) {
 
         @Override
         public SourceResponse call() throws Exception {
-            final SourceResponse sourceResponse =
-                    source.query(new QueryRequestImpl(request.getQuery(), request.getProperties()));
+            QueryRequest queryRequest;
+            if (CACHE_QUERY_MODE.equals(request.getPropertyValue(QUERY_MODE))) {
```

Comment:
I think this should also be done if the mode is INDEX_QUERY_MODE.

https://github.com/gordocanchola/ddf/blob/7f4f5f30ba971fff0ac86e4227fd3d2cf55837d1/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java#L106-L109